src/lib/pages/backoffice/DocumentRequest/DocumentRequestDetails/DocumentRequestActions/DeclineAction.js

```diff
@@ -0,0 +1,133 @@
+import { documentApi } from '@api/documents';
+import { ESSelectorModal } from '@modules/ESSelector';
+import { serializeDocument } from '@modules/ESSelector/serializer';
+import get from 'lodash/get';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Confirm, Dropdown } from 'semantic-ui-react';
+
+export class DeclineAction extends React.Component {
```

Comment:
nit: I find it a bit inconsistent that we have Action which is a component, not redux action (in the name)